src/components/EResourceSections/AgreementsList.js

```diff
@@ -15,11 +15,14 @@ import CustomCoverageIcon from '../CustomCoverageIcon';
 import EResourceLink from '../EResourceLink';
 import EResourceType from '../EResourceType';
 import { getResourceFromEntitlement, urls } from '../utilities';
-import { resourceClasses } from '../../constants';
-
-const AgreementsList = ({ eresource, id, isRelatedEntitlement, resources }) => {
-  const isTitle = eresource.type !== undefined;
 
+const AgreementsList = (
```

Comment:
Minor but trying to keep everything straight here since we now are gonna have like three or four components that render lists of agreements:
- Can we call this something like `EntitlementAgreementsList` or something? Just to underscore that you're actually passing in _entitlements_ and not agreements?
- On that point, can you call the prop `entitlements` instead of `resources`? And maybe use `e` instead of `res` in the arrow functions? Just to keep things clear when we look at this code again in a year.
- Not sure...but maybe at that point drag this out into its own folder at `src/components` rather than inside `EResourceSections`. The `Sections` folders should mostly be only filled with non-reusable components that make up a view.

Reply:
Done...Also added a boolean within the Agreements component wether to render the `relatedEntitlements` or not. Had to do this, if not the parent Accordion wasnt toggling as expected as `relatedEntitlements` call was bypassing the component containing  the `Accordion` .